Dockerfile-py2.7

```diff
@@ -25,11 +25,14 @@ ENV DISPLAY :99
 
 WORKDIR /workspace/Qt.py
 ENTRYPOINT cp -r /Qt.py /workspace && \
-    python build_caveats_tests.py && \
+    python build_caveats.py && \
+    python build_membership.py && \
 	Xvfb :99 -screen 0 1024x768x16 2>/dev/null & \
-	sleep 3 && \
+    while ! ps aux | grep -q '[0]:00 Xvfb :99 -screen 0 1024x768x16'; do echo "Waiting for Xvfb..."; sleep 1; done && \
```

Comment:
> I could make the search string include that last pipe bit. I didn't do it just to shorten down the line length.

On second thought, I can't. `ps` doesn't show piping:

    root         6  0.7  1.5 207336 31256 ?        Sl   10:05   0:00 Xvfb :99 -screen 0 1024x768x16

So this is fine.